Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -396,14 +409,48 @@ instance instLinearOrderedCommGroupWithZero [LinearOrderedCommGroup α] :
   __ := instLinearOrderedCommMonoidWithZero
   __ := commGroupWithZero
 
-end WithZero
+theorem one_lt_div' [LinearOrderedCommGroupWithZero α] (a : α) {b : α} (hb : b ≠ 0) :
+    1 < a / b ↔ b < a := by
+  rw [← mul_lt_mul_right (zero_lt_iff.mpr hb), one_mul, div_eq_mul_inv, inv_mul_cancel_right₀ hb]
```

Comment:
Can you generalise that to `GroupWithZero` + `LinearOrder` + compatibility typeclasses by assuming 
```suggestion
theorem one_lt_div' [LinearOrderedCommGroupWithZero α] (a : α) {b : α} (hb : 0 < b) :
    1 < a / b ↔ b < a := by
  rw [← mul_lt_mul_right (zero_lt_iff.mpr hb), one_mul, div_eq_mul_inv, inv_mul_cancel_right₀ hb]
```
This way it will also apply to eg linearly ordered fields. Same for the three lemmas below

Reply:
I managed to generalise the first but the other two landed me in a typeclass nightmare... I'd stick to this one for the moment (but will have a look at the import after lunch).